src/components/Article-buttons.jsx

```diff
@@ -0,0 +1,106 @@
+import Button from "@mui/material/Button";
+import { IconButton } from "@mui/material";
+import ButtonGroup from "@mui/material/ButtonGroup";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import ArrowCircleDownOutlinedIcon from "@mui/icons-material/ArrowCircleDownOutlined";
+import ArrowCircleUpOutlinedIcon from "@mui/icons-material/ArrowCircleUpOutlined";
+import { updateArticleVotes } from "../utils/api";
+import { useState } from "react";
+
+const ArticleButtons = ({ article, openComments, showComments }) => {
+  const [votes, setVotes] = useState(article.votes);
```

Comment:
because you're setting the state as the article.votes, if the user is on slow 3G or there are multiple users, your page is too slow to actually optimistically render the votes. Give it a try yourself throttling the network and vote up and down a few times. You can fix this by creating a variable of article.votes, and then setting that to state, so that the state isn't having to wait a second to find out what article.votes is and causing a delay